fwl-expect/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "fwl-expect",
+  "version": "1.0.0",
+  "description": "This is a jest-expect-message moded version for Sparta Courses",
+  "main": "index.js",
+  "repository": "git@github.com:fewlines-education/fwl-expect.git",
+  "author": "Fewlines",
+  "license": "MIT",
+  "private": false,
+  "dependencies": {
```

Comment:
Please note that `dependencies` in a package will be installed when you install the package.
In other cases, it's not that important to distinguish between deps and devDeps but for a package, it is :)

Reply:
So any `eslint` packages is moved to devDependecies? 🤔

Reply:
So we don't need all of those packages in the dependancies.
Tell me if I wrong @Fenntasy: We would only need jest as a deps and the rest in devDeps since we export a compiled version.